### ISSUE TITLE
error with cactus-prepare with custom sequences and Cromwell

### ISSUE THREAD

**Issue by lassancejm** [ComparativeGenomicsToolkit/cactus#241]
Hi, 

I am encountering an error when trying to generate an alignment with cactus in the step-by-step mode using cromwell. I should mention that I do not have problem with the example dataset, which ran to completion.

Following the instructions, I generated the workflow file like so:
`cactus-prepare cactus_input.txt --wdl >cactus.wdl`

Then ran Cromwell using this command:
`java -jar $APPS/cromwell/cromwell-50.jar run cactus.wdl`

The run stops pretty rapidly with the following error :
partial cromwell log:
```
[2020-05-15 11:18:17,41] [info] WorkflowManagerActor Workflow 1df10abe-1bdb-4175-ab5d-9ea34d2fd965 failed (during MaterializingWorkflowDescriptorState): cromwell.engine.workflow.lifecycle.materialization.MaterializeWorkflowDescriptorActor$$anon$1: Workflow input processing failed:
ERROR: Unexpected symbol (line 94, col 14) when parsing 'type_e'.

Expected identifier, got "{".

        File {}="AgoutiBAC_fa"
             ^

$type_e = :type <=> :lsquare $_gen22 :rsquare -> Type( name=$0, subtype=$2 )
 ```

I think there is something missing in the file produced by cactus-prepare. Specifically, the log from Cromwell suggests that the info are missing from some of the following lines in cactus.wdl:

```
workflow cactus_prepared {

    input {
        File seq_file="/n/holylfs03/LABS/hoekstra_lab/Users/lassance/Projects/cactus/cactus_input.txt"
        File? config_file
        File {}="BAC_fa"
        File {}="BEAST1_fa"
        File {}="BEAST2_fa"
    }
```   
In cactus_input.txt, I provide full paths to the fasta files corresponding to the 3 genomes to align. 

Any idea of what could be causing this? 

Thanks!

**Comment by glennhickey** [comment]
Yeah, something seems terribly wrong there.  Could you share the
cactus_input.txt file?  Also, if you're not using the current master branch
from cactus, that may be worth a try, as cactus-prepare has been fixed up
substantially since the last release.

**Comment by lassancejm** [comment]
I reinstalled cactus yesterday, but will do a quick check of the master branch. 

Here is the file.
[cactus_input.txt](https://github.com/ComparativeGenomicsToolkit/cactus/files/4636110/cactus_input.txt)

**Comment by glennhickey** [comment]
I can reproduce it, thanks.  Will push in a fix soon.  The automated tests
are only running with url inputs and it seems local file paths got broken
somehow.

**Comment by lassancejm** [comment]
Thanks for the update!

**Comment by glennhickey** [comment]
It should be corrected in #242, where running that command gives the following workflow input
```
workflow cactus_prepared {

    input {
        File seq_file="/home/hickey/dev/cactus/deermouse.txt"
        File? config_file
        File AgoutiBAC_fa="/n/holylfs03/LABS/hoekstra_lab/Users/lassance/Projects/Ppolionotus_oldcapture_reanalysis/cactus/Pman_agouti_bac.fa"
        File OLDFIELD_fa="/n/holylfs03/LABS/hoekstra_lab/Lab/PUBLIC/GENOMES/Ppol1.3_chr_NCBI/Ppol_1.3.3.chromosomes-unplaced_scaffolds.softmasked.fasta"
        File DEERMOUSE_fa="/n/holylfs03/LABS/hoekstra_lab/Lab/PUBLIC/GENOMES/Pman2.1_chr_NCBI/Pman2.1.3.chromosomes-unplaced_scaffolds.softmasked.fasta"
    }

```

**Comment by lassancejm** [comment]
Sweet, pipeline running now :-) 

Will let you know if I encounter other issues
